wp-content/themes/eighshift-task/src/admin/class-ajax-post-load.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * The Ajax_Post_Load specific functionality.
+ *
+ * @since   1.0.0
+ * @package Eighshift_Task\Admin
+ */
+
+namespace Eighshift_Task\Admin;
+
+use Eightshift_Libs\Core\Service;
+
+/**
+ * Class Ajax_Post_Load
+ *
+ * Class that handles load_posts AJAX action.
+ *
+ * @since 1.0.0
+ */
+class Ajax_Post_Load implements Service {
```

Comment:
this is okej for practice. But you could have used Rest for this and not a custom implementation. But ok for practice.

We are structuring the code by features and not by location of use (agreed few wp standups ago), so this is a logical individual part of a feature. It should go to its own folder.

check here for reference:
https://github.com/infinum/wordpress-infinum-web/tree/staging/wp-content/themes/infinum/src

Reply:
👍